packages/circuit-ui/components/SelectorGroup/SelectorGroup.stories.tsx

```diff
@@ -70,4 +70,26 @@ export const Multiple = (args: SelectorGroupProps) => {
   );
 };
 
-Multiple.args = { ...baseArgs, multiple: true };
+MultipleSelection.args = { ...baseArgs, multiple: true };
+
+export const MultipleLines = () => {
```

Comment:
I like the idea of adding a new story for this use case. I wonder if we can make it a bit more realistic though. An advanced use case that I've seen a couple of times on the dashboard features multi-line labels:

<img width="867" alt="Screenshot 2022-07-08 at 13 53 52" src="https://user-images.githubusercontent.com/11017722/177987144-5795978c-ff23-4686-8add-7be62b47a855.png">

Replicating this use case would enable us to test the styles for different heights _and_ the styles for multiple elements inside the label. WDYT?

Reply:
I believe that the current story is simple and shows clearly what's the behavior when there is content with different heights on different selectors, but I'll gladly work on a more ellaborated story if you think it's a good idea :)

Reply:
Initially I was also thinking "let's make the example closer to this real use-case" but since this is not great a11y-wise (labels are read without semantics and should be clear and concise[^1]), I think we can keep the simpler version.

But I'll probably just extend the base story instead of duplicating it with an extra option

[^1]: so the top-right selector on the screenshot would read `Direct to bank account your funds will be deposited in your bank account as soon as they become available, arriving in your account in 2 dash 3 business days, tickbox`. Very verbose

Reply:
Done in cbd60b7